src/sgame/sg_client.cpp

```diff
@@ -1377,11 +1377,17 @@ void ClientBegin( int clientNum )
 		switch ( g_showHelpOnConnection.Get() )
 		{
 		case 0:
-			if (0)
-		default:
-			if ( !client->pers.admin )
+			break;
 		case 1:
 			G_TriggerMenu( client->ps.clientNum, MN_WELCOME );
+			break;
+		case 2:
+		default:
```

Comment:
We shouldn't need a default since there are no other possible values.

Reply:
I wonder if keeping a switch here is really a good idea.
From what I understand, all that code means:

```
if ( ! ( g_showHelpOnConnection.Get() == 2 && client->pers.admin ) )
{
    G_TriggerMenu( client->ps.clientNum, MN_WELCOME );
}
```

which I consider more readable.